sphinx/writers/html5.py

```diff
@@ -0,0 +1,958 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.writers.html5
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Experimental docutils writers for HTML5 handling Sphinx' custom nodes.
+
+    This class is an almost copy of HTMLTranslator because:
```

Comment:
Is this comment needed? Certainly, I requested the difference of writers. But it is only used for reviewing.
I worried about this comment will not be maintained in future version.

This comment describes differences with HTML4 writer mainly. But the old writer will be deprecated in nearly future. I think it is not a description of this codes, and it is used temporarily.
Therefore it would be better to use github-comments or other temporarily space, not in source code.